hubble/__init__.py

```diff
@@ -39,7 +39,15 @@ def foo():
     @wraps(func)
     def arg_wrapper(*args, **kwargs):
         if Client(jsonify=True).token:
-            return func(*args, **kwargs)
+            try:
+                Client(jsonify=True).get_user_info()
+                return func(*args, **kwargs)
+            except AuthenticationRequiredError:
+                raise AuthenticationFailedError(
+                    response={},
+                    message=f'Jina auth token has expired. {func!r} requires login to Jina AI, '
+                    f'please do `jina auth login -f` or set env variable `JINA_AUTH_TOKEN` with updated token',
+                ) from None
```

Comment:
Should we use `get_user_info` at the first place? No need to check if `token` exists or not.

Reply:
Yeah, that makes more sense. Thanks for the suggestion. Have updated the PR accordingly.